python/fapolicy_analyzer/tests/test_main.py

```diff
@@ -0,0 +1,14 @@
+import context  # noqa: F401
+import sys
+from unittest.mock import patch
+from ui.__main__ import parse_cmdline, gbAutosaveEnabled, gstrEditSessionFileName
+
+
+def test_parse_args(mocker):
+    testargs = ["prog", "-v", "-a", "-s", "/tmp/TmpFileTemplate.tmp"]
+    with patch.object(sys, 'argv', testargs):
+        args = parse_cmdline()
+        print(gbAutosaveEnabled, gstrEditSessionFileName)
+        assert args["verbose"]
+        assert args["autosave"]
+        assert args["session"] == "/tmp/TmpFileTemplate.tmp"
```

Comment:
Can we test this by calling `main()` with some command line args and then testing the `stateManager` is set properly.  That's really what we care about, that in the end the `stateManager` is right.  Here's a link that might help with how to pass args to the `main` method https://stackoverflow.com/a/45175102/4351673

Reply:
Sure. I think that can be done.

Reply:
I should probably do both, i.e. it is unit testing. But I agree main() should be tested if possible.

Reply:
Done. main90 is invoked both w/ and w/o all supported options.